src/components/NavBar.vue

```diff
@@ -29,6 +29,14 @@
       />
       <v-spacer />
 
+      <amb-button
+        data-cy="timeline-icon"
+        is-icon
+        icon="history"
```

Comment:
@vickywane the icon does not show when I test . There is a visual slot in the NavBar that reacts to mouseover and click, but the icon is not visible. Something is off with the mapping from names to graphics.

![Screen Shot 2021-03-12 at 1 32 40 PM](https://user-images.githubusercontent.com/2234901/110989426-7e47b280-8337-11eb-8e92-084f346a27f7.png)
![Screen Shot 2021-03-12 at 1 28 49 PM](https://user-images.githubusercontent.com/2234901/110989429-7e47b280-8337-11eb-8f7e-0c8bc1fcae12.png)
![Screen Shot 2021-03-12 at 1 28 23 PM](https://user-images.githubusercontent.com/2234901/110989430-7ee04900-8337-11eb-9565-5d2efe084ce0.png)